src/pudl/transform/ferc714.py

```diff
@@ -306,6 +350,8 @@ def _post_process(df: pd.DataFrame, table_name: str) -> pd.DataFrame:
     Applies standard data types and ensures that the tables generally conform to the
     schemas we have defined for them.
 
+    TODO: rip this out. enforce_schema happens via the io_managers now.
+
```

Comment:
What's stopping us from removing it now?

Reply:
on this branch its being called in the annual table! so i wanted to leave this note to remind myself to rip it out once we get all these tables merged into one branch

Reply:
We should create a list somewhere. I'll add it to your original hourly PR